packages/protocol-http/src/validHostname.ts

```diff
@@ -0,0 +1,4 @@
+export function validHostname(hostname: string): boolean {
```

Comment:
use `isValidHostname`
```suggestion
export function isValidHostname(hostname: string): boolean {
```

StackOverflow discussion https://softwareengineering.stackexchange.com/questions/348649

Reply:
Fixed.